network-api/networkapi/templates/pages/campaign-opportunity-base.html

```diff
@@ -0,0 +1,59 @@
+{% extends "./base-compiled.html" %}
+
+{% block bodyID %}{{ suffix }}{% endblock %}
+
+{% block content %}
+  <div class="container">
+    <div class="row">
+      <div class="py-5 {% if page.parent.title != "{{ parentTitle }}" or page.children.all.count > 0 %}col-12 col-md-8 offset-md-3{% else %}col-12 col-lg-8 offset-lg-2{% endif %}">
+        <p class="h5-black">{{ page.parent.title }}</p>
+        <h1 class="h1-white hidden-sm-down">{% block h1 %}{% endblock %}</h1>
+      </div>
+    </div>
+  </div>
+  <div class="hidden-md-up mb-5" id="multipage-nav-mobile">
+    <div class="container">
+      <div class="row">
+        <div class="col-12"></div>
+      </div>
+    </div>
+  </div>
+  <div class="container">
+    <div class="row mb-5">
+      {% if page.parent.title != "{{ parentTitle }}" or page.children.all.count > 0 %}
+      <div class="col-12 col-md-3 mb-3 hidden-sm-down">
+        <div class="multipage-nav" id="multipage-nav">
+          {% load pages_tags %}
+          {% page_menu "pages/menus/side-menu-items.html" %}
+        </div>
+      </div>
+      {% endif %}
+
+      {% block subcontent %}
+      <div class="{% if page.parent.title != "{{ parentTitle }}" or page.children.all.count > 0 %}col-12 col-md-9{% else %}col-12 col-lg-8 offset-lg-2{% endif %}">
+        <div class="row">
+          <div class="cms col-12">{{ page.landingpage.content | safe }}</div>
+        </div>
+      </div>
+      {% endblock %}
+    </div>
+
+    {% if page.landingpage.signup %}
```

Comment:
Campaign pages don't have a `signup`, they have `petition`. Should this block be in the landingpage specific template?